src/pages/informational/WpPlugin.tsx

```diff
@@ -0,0 +1,225 @@
+import character from "assets/laira/laira-waiving.png";
+import Copier from "components/Copier";
+import Image from "components/Image";
+
+export default function WpPlugin() {
+  return (
+    <div
+      className={`padded-container mt-16 grid content-start pb-16 @container`}
```

Comment:
no need for backticks if not using templates `${}`
```suggestion
      className="padded-container mt-16 grid content-start pb-16 @container"
```